packages/@react-aria/list/docs/useList.mdx

```diff
@@ -0,0 +1,439 @@
+
+  return (
+    <ul
+      {...gridProps}
+      ref={ref}
+      style={{
+        padding: 0,
+        margin: '5px 0',
+        listStyle: 'none',
+        border: '1px solid gray',
+        maxWidth: 400
+      }}
+    >
+      {[...collection].map((item) => (
+        <ListItem key={item.key} item={item} state={state} />
+      ))}
+    </ul>
+  );
+}
+
+function ListItem({ item, state }) {
+  let ref = React.useRef();
+
+  let {
+    rowProps,
+    gridCellProps,
+    isPressed,
+    isSelected,
+    isFocused,
+    isDisabled
+  } = useListItem(
+    {
+      node: item
+    },
+    state,
+    ref
+  );
+  let {isFocusVisible, focusProps} = useFocusRing();
+  let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
+
+  let backgroundColor;
+  let color = "black";
+
+  if (isSelected) {
+    backgroundColor = "blueviolet";
+    color = "white";
+  } else if (isFocused) {
+    backgroundColor = "gray";
+  } else if (isDisabled) {
+    backgroundColor = "transparent";
+    color = "gray";
+  }
+
+  return (
+    <li
+      {...mergeProps(rowProps, focusProps)}
+      ref={ref}
+      style={{
+        background: isSelected
+          ? 'blueviolet'
+          : isPressed
+            ? 'var(--spectrum-global-color-gray-400)'
+            : item.index % 2
+              ? 'var(--spectrum-alias-highlight-hover)'
+              : 'none',
+        color: isSelected ? 'white' : null,
+        padding: '2px 5px',
+        outline: isFocusVisible ? '2px solid orange' : 'none'
+      }}
+    >
+      <div {...gridCellProps}>
+        {showCheckbox && <ListCheckbox item={item} state={state} />}
+        {item.rendered}
+        <span style={{float: 'right'}}>
+          <Button onPress={() => alert(`Info for item ${item.key}`)}>Info</Button>
+        </span>
+      </div>
+    </li>
+  );
+}
+
+<List aria-label="Example List" selectionMode="multiple" selectionBehavior="replace">
+  <Item key="1">Charizard</Item>
+  <Item key="2">Blastoise</Item>
+  <Item key="3">Venusaur</Item>
+  <Item key="4">Pikachu</Item>
+</List>
+```
+
+### Adding selection checkboxes
+
+Next, let's add support for selection checkboxes. For multiple selection, we'll want to add a checkbox to the row to allow the user to select rows.
+This is done using the <TypeLink links={docs.links} type={docs.exports.useListSelectionCheckbox} />
+hook. It is passed the `parentKey` of the cell, which refers to the row the cell is contained within. When the user
+checks or unchecks the checkbox, the row will be added or removed from the List's selection.
+
+In this example, we pass the result of the `checkboxProps` into the <TypeLink links={checkboxDocs.links} type={checkboxDocs.exports.useCheckbox} />
+hook and render an `<input>` element directly, but it's likely you'll have a `Checkbox` component in your component library that uses these hooks already.
+See the [useCheckbox docs](useCheckbox.html) for more information.
```

Comment:
Need to update this paragraph now that you've moved this to a separate component, e.g. "The `Checkbox` component used in this example is independent, and can be used separately from `List`. See below for the code." etc.